src/styles/box/stars.scss

```diff
@@ -0,0 +1,31 @@
+.product-card__stars {
+  width: 100px;
+  display: flex;
+  height: $star-height;
+  align-items: center;
+  justify-content: left;
+  list-style: none;
+  padding: 0;
+  margin: 0 auto;
+}
+
+.product-card__star {
+  display: flex;
+  background-image: $star-bg;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  box-sizing: border-box;
+  width: $star-width;
+  height: $star-height;
+  padding: 0;
+  margin-right: 6px;
+  margin-left: 1px;
+}
+
+.product-card__star:nth-child(-n + 4) {
+  background-image: $star-active-bg;
```

Comment:
The variable `$star-active-bg` is used here, but it is not defined in this file. Ensure that `$star-active-bg` is defined in a file that is imported before this one, or define it here if it's missing.